src/Symfony/Component/Messenger/Handler/HandlersLocator.php

```diff
@@ -20,9 +20,9 @@
  * @author Nicolas Grekas <p@tchwork.com>
  * @author Samuel Roze <samuel.roze@gmail.com>
  */
-class HandlersLocator implements HandlersLocatorInterface
+abstract class HandlersLocator implements HandlersLocatorInterface
```

Comment:
This is quite an important BC break, and I am unsure how to resolve it.

One of the test is actually failing on php (8.0, low-deps) because it tries to instantiate this class.

What would be a good method to keep backward compatibility here?